pkg/network/render.go

```diff
@@ -353,6 +354,8 @@ func isNetworkChangeSafe(prev, next *operv1.NetworkSpec, infraRes *bootstrap.Inf
 		if !isSupportedDualStackPlatform(infraRes.PlatformType) {
 			return errors.Errorf("%s is not one of the supported platforms for dual stack (%s)", infraRes.PlatformType,
 				strings.Join(dualStackPlatforms.List(), ", "))
+		} else if string(configv1.OpenStackPlatformType) == string(infraRes.PlatformType) {
```

Comment:
We could probably just split `isSupportedDualStackPlatform()` into another function to be used here, but no big deal if it's just OpenStack that doesn't support migrations.